platform/NonCopyable.h

```diff
@@ -211,18 +211,17 @@ class NonCopyable {
     }
 
 #else
-private:
     /**
      * Declare copy constructor as private. Any attempt to copy construct
      * a NonCopyable will fail at compile time.
      */
-    NonCopyable(const NonCopyable &);
+    NonCopyable(const NonCopyable &) = delete;
 
     /**
      * Declare copy assignment operator as private. Any attempt to copy assign
```

Comment:
Already fixed, following previous comment.

Reply:
My fault. I tried to do this on a phone and stumbled mightily over myself. I made the suggested changes yesterday, went home and checked this morning to see my code comments weren’t submitted. Either I timed it just right so that you pushed your changes right before I actually hit submit, or I flat out missed it. 

Thanks!